File: packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCS/ReactPackageProvider.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using Microsoft.ReactNative;
using Microsoft.ReactNative.Managed;

namespace SampleLibraryCS
{
    public sealed class ReactPackageProvider : IReactPackageProvider
    {
        public void CreatePackage(IReactPackageBuilder packageBuilder)
        {
            packageBuilder.AddReflectionReactPackageProvider<ReactPackageProvider>();
        }
    }
}

File: packages/xaml-calendar-view/windows/XamlCalendarView/ReactPackageProvider.cpp
#include "pch.h"

#include "ReactPackageProvider.h"
#if __has_include("ReactPackageProvider.g.cpp")
#include "ReactPackageProvider.g.cpp"
#endif

#include "XamlCalendarView.h"

using namespace winrt::Microsoft::ReactNative;

namespace winrt::XamlCalendarView::implementation
{

void ReactPackageProvider::CreatePackage(IReactPackageBuilder const &packageBuilder) noexcept
{
  AddAttributedModules(packageBuilder, true);
  RegisterCalendarViewNativeComponent(packageBuilder);
  //settings.PackageProviders().Append(winrt::Microsoft::ReactNative::Xaml::ReactPackageProvider());
}

} // namespace winrt::XamlCalendarView::implementation
